include/vapor/GeoImageTMS.h

```diff
@@ -9,13 +9,78 @@
 #include <xtiffio.h>
 #include <geotiff.h>
 #endif
+#include <sstream>
+#include <fstream>
+#include <sys/stat.h>
 #include <vapor/MyBase.h>
 #include <vapor/UDUnitsClass.h>
 #include "GeoTileMercator.h"
 #include "GeoImage.h"
 
 namespace VAPoR {
 
+static inline bool IsTMSFile( 
```

Comment:
This function is trying to do too much. All that it needs to do is determine if the file is a TMS image data base. I.e. does it have a ".tms" extension. Also, our error policy is that functions that return booleans do not report errors as this function does.

Reply:
Fixed.